### DISCUSSION TITLE
Weight number does not change with slider

### DISCUSSION THREAD

**Discussion by suonnon** [muerrilla/stable-diffusion-NPW#8] [Q&A]
When I use cursor to move the slider, the weight number that is showing in the textbox does not change (actually it still affects the prompt but now showing the changed number on the UI).
![image](https://github.com/muerrilla/stable-diffusion-NPW/assets/133345496/b9fd5cdd-e8f5-45b3-b968-a3b8d39c87a6)
I'm using A1111 ver 1.4.0

**Comment by muerrilla** [comment]
There's an issue ( #7 ) about this, but I can't help unless one of you guys tell me the error, since I'm on an older version of the webui and can't replicate the problem.

**Reply by suonnon** [reply]
> There's an issue ( #7 ) about this, but I can't help unless one of you guys tell me the error, since I'm on an older version of the webui and can't replicate the problem.

So i'm the first one talk about this? Hope you could release the update soon
